const catchAsyncErrors = require("../middleware/catchAsyncErrors")
const User = require("../models/User")


exports.registerUser = catchAsyncErrors(async(req,res,next)=>{

})

exports.login = catchAsyncErrors(async(req,res,next)=>{

})

exports.logout = catchAsyncErrors(async(req,res,next)=>{

})

exports.addUser = catchAsyncErrors(async(req,res,next)=>{

})

exports.updateProfile = catchAsyncErrors(async(req,res,next)=>{

})

exports.getUserDetails = catchAsyncErrors(async(req,res,next)=>{

})

exports.deleteUser = catchAsyncErrors(async(req,res,next)=>{

})